openapi/ledger_spec.yaml

```diff
@@ -1281,6 +1281,10 @@ components:
     ExternalTransactionState:
       type: object
       properties:
+        can_retry:
+          type: boolean
+          default: false
```

Comment:
as per https://github.com/alphagov/pay-ledger/pull/2657/files#diff-9c8dd90f4e46450f6d1b524141213d176f435827db34b3705b60e704fff438caR58 default is null but false here

Reply:
👍